README.md

```diff
@@ -370,13 +557,17 @@ Closures provide great utility.
 ##### Demo - return new functions
 
 ```js
-const memoFactory = function (rememberMe) {
```

Comment:
Or maybe not. Justify this again please? I know we discussed it but I'd like your thoughts recorded.